src/grpc.ts

```diff
@@ -16,8 +21,20 @@ export default class GrpcClient {
         client_id: string,
         client_secret: string,
         refresh_token: string,
-        login_customer_id?: string
+        login_customer_id?: string,
+        gads_node_options?: GoogleAdsNodeOptions
     ) {
+
+        const additional_options: any = {}
+
+        // Apply google-ads-node options if specified
+        if(gads_node_options?.prevent_mutations) {
```

Comment:
I don't think that we should expose `google ads node` from this library, especially via a cryptic `gads_node_options` (pretty confusing if read without context). Why not just use put `logging` and `prevent_mutations` on the base options object? Or perhaps rename `gads_node_options` to `grpc_options`?

Reply:
Yes, that's a better idea, and much easier to understand as you say 👍